tests/main/hook-permissions/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: Ensure that post-refresh hook has permissions for its connected slot.
+
+# Note: it is critical for this test to use a slot that requires
+# peer=<plug security tag> expression label for dbus, and not simply
+# label=unconfined on a permanent slot snippet.
+details: |
+    Ensure that post-refresh hook has permissions to query upower-observe slot
+    and enumerate upower devices.
+
+systems: [-fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*]
+
+restore: |
+    rm -f ./enumerate.txt
```

Comment:
this is done automaticallly, so it is not needed.

Reply:
Thanks, removed.